package.json

```diff
@@ -5,7 +5,7 @@
     "scripts": {
         "clean": "lerna run --parallel clean",
         "clean:portal": "lerna run clean:portal --stream",
-        "build": "yarn patch-package && lerna run --ignore @fremtind/browserslist-config-jkl build --stream",
+        "build": "yarn patch-package && lerna run --ignore @fremtind/browserslist-config-jkl --ignore @fremtind/portal build --stream",
```

Comment:
Er det intensjonen å ta vekk portal bygget?

Reply:
Ja, det kom med som en bieffekt av at vi endret navn på byggscriptet der for en stund siden. Siden det kjøres i parallell bygger den nå før alle komponentene er ferdig bygget, så bygget blir litt derpy (i hvert fall lokalt). Bygges uansett igjen før publisering/testing.